mis_builder/models/mis_report.py

```diff
@@ -892,18 +904,14 @@ def declare_and_compute_period(self, kpi_matrix,
                                             query argument and returns a
                                             domain compatible with the query
                                             underlying model
+        :param locals_dict: personalized localsdict
```

Comment:
personalized locals dictionary used as evaluation context for the KPI expressions